### PR TITLE
24 create the header section

### LINKED ISSUE #24: Create the header section
**ACCEPTANCE CRITERIA**

Let's start by creating the header section of the website based on your design references

https://www.figma.com/design/ZuyFE2DsQCsxOu8woyTPrn/isela-erwan?node-id=0-1&t=q4aTwALe68HsLqyP-1

**TASKS**

- [x] Make sure you create a new branch from develop using the new branch creation feature that you can find on the right sidebar of this issue
- [x] Create the header html component with the correct semantic (this will be an organism in the component structure)
- [x] Create the styling related to it
- [x] Make sure it is responsive
- [x] Submit a PR from your feature branch against develop

### PR DESCRIPTION
**Pull request by Isynopoulos1**
-I added responsive rules.
-A hamburguer menu appears when it's in mobile size.
-I styled the nav and I established constants

### CODE REVIEW

File: src/components/atoms/link/Link.styles.jsx
Comment (erwanriou) [inline src/components/atoms/link/Link.styles.jsx:1]: I tested locally and i think you need a cursor pointer style here

File: src/components/molecules/header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+} from "./Header.styles";
+
+// IMPORT ASSETS
+import { languages } from "@utils";
+
+const Header = () => {
+  // HOOKS
+  const [isMobile, setIsMobile] = useState(false);
+
+  useEffect(() => {
```
Comment (erwanriou) [inline src/components/molecules/header/Header.jsx:31]: This `useEffect` should be in the `// LIFECYLES` section.

File: src/components/molecules/header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      <HeaderContainer>
+        <HeaderLeftContainer>
+          <Logo />
+          <nav>
+            {links.map(link => (
+              <Link label={link.label} onClick={() => handleLink(link.href)} />
+            ))}
+          </nav>
+        </HeaderLeftContainer>
+        {!isMobile && (
```
Comment (erwanriou) [inline src/components/molecules/header/Header.jsx:97]: Why only hide the language menu. Would it not be better to hide it all and put all the menu elements inside the future burger? Also in responsive when i click on the burger nothing happen. I don't see a menu that appear.

You are missing an important part of the functionality here.

File: src/components/molecules/header/Header.styles.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+`;
+
+export const HeaderContainer = styled.div`
+  position: relative;
+  max-width: ${contentW.lg};
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
Comment (erwanriou) [inline src/components/molecules/header/Header.styles.jsx:21]: There is something wrong in your responsive after `1500px` 
Screenshot from 2024-06-07 17-04-42

As you can see there is no space between the logo and the screen. That sound incorrect. Same for the otherside

**Review by erwanriou — CHANGES_REQUESTED** [review]
Good job but still some missing pieces of functionalities.

File: src/components/atoms/link/Link.styles.jsx
Comment (Isynopoulos1) [reply src/components/atoms/link/Link.styles.jsx:1]: done

File: src/components/molecules/header/Header.jsx (hunk 1 above)
Comment (Isynopoulos1) [reply src/components/molecules/header/Header.jsx:31]: done

File: src/components/molecules/header/Header.jsx (hunk 2 above)
Comment (Isynopoulos1) [reply src/components/molecules/header/Header.jsx:97]: done

File: src/components/molecules/header/Header.styles.jsx (hunk 3 above)
Comment (Isynopoulos1) [reply src/components/molecules/header/Header.styles.jsx:21]: done

**Comment by Isynopoulos1** [comment]
I fixed the issues also I added the functionality to the Header. Is ready to review

**Review by erwanriou — APPROVED** [review]
Nice job! You can merge it

**Pull request merged**